demisto_sdk/commands/validate/README.md

```diff
@@ -1,154 +1,100 @@
+* **--skip-new-validate**
+Wether to run the new validate flow.
 
 **Examples**:
-`demisto-sdk validate -g --no-backwards-comp`
-This will validate only changed files from content origin/master branch and will exclude backwards
-compatibility checks.
-<br><br>
 
 `demisto-sdk validate --prev-ver SHA1-HASH`
 This will validate only changed files from the branch given (SHA1).
-<br><br>
 
 `demisto-sdk validate --post-commit`
 This indicates that the command runs post commit.
-<br><br>
 
 `demisto-sdk validate -i Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml`
 This will validate the file Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml only.
-<br><br>
 
 `demisto-sdk validate -a`
-This will validate all files under `Packs` directory
-<br><br>
+This will validate all files under `Packs` directory.
 
 `demisto-sdk validate -i Packs/HelloWorld`
-This will validate all files under the content pack `HelloWorld`
-<br><br>
+This will validate all files under the content pack `HelloWorld`.
 
-`demisto-sdk validate -i Packs/HelloWorld --run-specific-validations BA101`
-This will validate all files under the content pack `HelloWorld` using only the validation corresponds to the error code BA101.
-<br><br>
-
-`demisto-sdk validate -i Packs/HelloWorld --run-specific-validations BA`
-This will validate all files under the content pack `HelloWorld` using only the validations from error type of BA.
-<br><br>
-
-### Notes
-* In external repositories (repos which contain the `.private-repo-settings` file in its root) **all** the validations are ignorable.
+`demisto-sdk validate --run-old-validate --skip-new-validate -a`
+This will validate all files in the repo using the old validate method.
 
+`demisto-sdk validate --config-path {config_file_path} -a`
+This will validate all files in the repo using the settings configured in the config file in the given path.
 
 ### Error Codes and Ignoring Them
-Starting in version 1.0.9 of Demisto-SDK, each error found by validate (excluding `pykwalify` errors) has an error
-code attached to it - the code can be found in brackets preceding the error itself.
+Each error found by validate  has an error code attached to it - the code can be found in brackets preceding the error itself.
 For example: `path/to/file: [IN103] - The type field of the proxy parameter should be 8`
+In addition, each pack has a `.pack-ignore` file. In order to ignore a certain validation for a given file, the error-code needs to be listed in the **ignorable_errors** section in the config-file (more about this in the config file section), and the user need to mention the file name (only the name and extension, without the whole path), and the error code to ignore.
+For example: This .pack-ignore will not fail ipinfo_v2.yml on the validations with the codes BA108 & BA109.
+[file:ipinfo_v2.yml]
+ignore=BA108,BA109
+
+### Config file
+Each user will have a personal config file which he can edit however he wants.
```

Comment:
we need to also give an example config file which is complex, so not the default config

Reply:
The default config includes all the features but just with a small amount of validations